vgaidarji/github-actions-pr-merger
# Migrate from checks to commit statuses

### What has been done
1. Migrated from checks to commit statuses


### How to test
1. Merge PR
2. Leave a comment on PR and check action build result
3. Confirm last PR commit status updated
